packages/filters/filter-displacement/src/DisplacementFilter.ts

```diff
@@ -63,15 +68,17 @@ export class DisplacementFilter extends Filter
      * @param {PIXI.RenderTexture} output - The output target.
      * @param {PIXI.CLEAR_MODES} clearMode - clearMode.
      */
-    apply(filterManager, input, output, clearMode)
+    public apply(
+        filterManager: systems.FilterSystem, input: RenderTexture, output: RenderTexture, clearMode: CLEAR_MODES
+    ): void
     {
         // fill maskMatrix with _normalized sprite texture coords_
         this.uniforms.filterMatrix = filterManager.calculateSpriteMatrix(this.maskMatrix, this.maskSprite);
         this.uniforms.scale.x = this.scale.x;
         this.uniforms.scale.y = this.scale.y;
 
         // Extract rotation from world transform
-        const wt = this.maskSprite.transform.worldTransform;
+        const wt = this.maskSprite.worldTransform;
```

Comment:
switching to a property? OK, I don't see a problem, filters are heavy anyway :) Oh, its because of my interface. Its fine, I agree with those changes.